sdk/core/azure-core/test/ut/operation.cpp

```diff
@@ -88,3 +88,22 @@ TEST(Operation, Status)
   EXPECT_THROW(operation.Value(), std::runtime_error);
   EXPECT_EQ(operation.Status(), OperationStatus::Cancelled);
 }
+
+TEST(Operation, ResumeToken)
+{
+  StringClient client;
+  std::string token;
+  {
+    auto operation = client.StartStringUpdate();
+    token = operation.GetResumeToken();
+  }
+  {
+    for (auto resumedOperation = StringOperation::CreateFromResumeToken(client, token);
```

Comment:
nit: consider adding a test with passing in context and also cancelling the context to verify `CreateFromResumeToken` throws the right exception

Reply:
There's no context for this unit test as no network I/O is involved.  
If you want to improve this unit test with a context, create a ticket for it and it can be done as a new PR